.github/workflows/test.yaml

```diff
@@ -0,0 +1,74 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - '**/*'
+
+jobs:
+  test-android-sdk:
+    runs-on: macos-latest
+    env:
+      ANDROID_API_LEVEL: 33
+    steps:
+      - name: Check out Java SDK
+        uses: actions/checkout@v3
+        with:
+          repository: 'Eppo-exp/android-sdk'
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+
+      - name: 'Set up GCP SDK'
+        uses: 'google-github-actions/setup-gcloud@v0'
+
+      - name: Restore gradle.properties
+        env:
+          MAVEN_USERNAME: ${{ secrets.MAVEN_USERNAME }}
+          MAVEN_PASSWORD: ${{ secrets.MAVEN_PASSWORD }}
```

Comment:
It's ok these aren't populated in this repo, they just need to be defined (even if empty string)